client/web/src/jscontext.ts

```diff
@@ -7,7 +7,15 @@ export type DeployType = 'kubernetes' | 'docker-container' | 'docker-compose' |
  */
 
 export interface AuthProvider {
-    serviceType: 'github' | 'gitlab' | 'http-header' | 'openidconnect' | 'sourcegraph-operator' | 'saml' | 'builtin'
+    serviceType:
+        | 'github'
+        | 'gitlab'
+        | 'bitbucketCloud'
```

Comment:
should we follow the existing convention and name it `bitbucket-cloud` instead?
```suggestion
        | 'bitbucket-cloud'
```

Reply:
This value actually comes from here: https://github.com/sourcegraph/sourcegraph/blob/e5124274ffe48ac1aaf1e9f7781b05e6aa1b93cb/internal/extsvc/types.go#L148:2

It's the service type for the code host connection. So it would have to be changed all the way up the stack, and we'd have possible backwards compatibility issues with existing code host connections.

Reply:
makes sense, disregard then